tests/db/schemas/mssql.sql

```diff
@@ -146,6 +162,44 @@ CREATE TABLE latest_metrics (
 )
 
 
+CREATE TABLE logged_model_metrics (
+	model_id VARCHAR(36) COLLATE "SQL_Latin1_General_CP1_CI_AS" NOT NULL,
+	metric_name VARCHAR(500) COLLATE "SQL_Latin1_General_CP1_CI_AS" NOT NULL,
+	metric_timestamp_ms BIGINT NOT NULL,
+	metric_step BIGINT NOT NULL,
+	metric_value FLOAT,
+	run_id VARCHAR(32) COLLATE "SQL_Latin1_General_CP1_CI_AS" NOT NULL,
+	dataset_uuid VARCHAR(36) COLLATE "SQL_Latin1_General_CP1_CI_AS",
+	dataset_name VARCHAR(500) COLLATE "SQL_Latin1_General_CP1_CI_AS",
+	dataset_digest VARCHAR(36) COLLATE "SQL_Latin1_General_CP1_CI_AS",
+	CONSTRAINT logged_model_metrics_pk PRIMARY KEY (model_id, metric_name, metric_timestamp_ms, metric_step, run_id),
+	CONSTRAINT "FK__logged_mo__model__797309D9" FOREIGN KEY(model_id) REFERENCES logged_models (model_id) ON DELETE CASCADE,
+	CONSTRAINT "FK__logged_mo__run_i__7A672E12" FOREIGN KEY(run_id) REFERENCES runs (run_uuid) ON DELETE CASCADE
+)
+
+
+CREATE TABLE logged_model_params (
+	model_id VARCHAR(36) COLLATE "SQL_Latin1_General_CP1_CI_AS" NOT NULL,
+	experiment_id INTEGER NOT NULL,
+	param_key VARCHAR(255) COLLATE "SQL_Latin1_General_CP1_CI_AS" NOT NULL,
+	param_value VARCHAR COLLATE "SQL_Latin1_General_CP1_CI_AS" NOT NULL,
+	CONSTRAINT logged_model_params_pk PRIMARY KEY (model_id, param_key),
+	CONSTRAINT "FK__logged_mo__exper__7D439ABD" FOREIGN KEY(experiment_id) REFERENCES experiments (experiment_id),
+	CONSTRAINT "FK__logged_mo__model__7E37BEF6" FOREIGN KEY(model_id) REFERENCES logged_models (model_id) ON DELETE CASCADE
+)
```

Comment:
The statements in this file is created by:

https://github.com/mlflow/mlflow/blob/a95ebd1443295792e92dff78b46ca8009777e123/tests/db/test_schema.py#L24-L40


The actual query executed by sqlalchemy is:

```
CREATE TABLE logged_model_params (
        model_id VARCHAR(36) NOT NULL, 
        experiment_id INTEGER NOT NULL, 
        param_key VARCHAR(255) NOT NULL, 
        param_value VARCHAR(max) NOT NULL, 
        CONSTRAINT logged_model_params_pk PRIMARY KEY (model_id, param_key), 
        FOREIGN KEY(experiment_id) REFERENCES experiments (experiment_id), 
        FOREIGN KEY(model_id) REFERENCES logged_models (model_id) ON DELETE CASCADE
)
```